### PR TITLE
feat: manage user's friends

### PR DESCRIPTION
**Pull request by prndrbr**
- [x] Include tests for bug fix and new functionality
- [x] Updated documentation for new feature

This allows to manage user's friends:
- get the followers 
- get, add or remove a following

Also fix a return type in the doc

### CODE REVIEW

File: src/deezer/client.py [hunk 1]
```diff
@@ -409,6 +410,46 @@ def remove_user_artist(self, artist_id: int) -> bool:
...
+                 of :class:`~deezer.User` instances.
+        """
+        user_id_str = str(user_id) if user_id else "me"
+        return self._get_paginated_list(f"user/{user_id_str}/followings")
+
+    def add_user_following(self, user_id: int) -> bool:
+        """
+        Add an user to the current user's followings
+
+        :param user_id: the ID of the user to add.
```
Comment (browniebroke) [inline src/deezer/client.py:439]:
```suggestion
        :param user_id: the ID of the user to follow.
```

File: src/deezer/client.py [hunk 2]
```diff
@@ -409,6 +410,46 @@ def remove_user_artist(self, artist_id: int) -> bool:
...
+        :return: boolean whether the operation succeeded.
+        """
+        return self.request("POST", "user/me/followings", user_id=user_id)
+
+    def remove_user_following(self, user_id: int) -> bool:
+        """
+        Remove an user from the current user's followings
+
+        :param user_id: the ID of the user to remove.
```
Comment (browniebroke) [inline src/deezer/client.py:448]:
```suggestion
        :param user_id: the ID of the user to stop following.
```

**Review by browniebroke — APPROVED** [review]
Nice

File: src/deezer/client.py [hunk 3]
```diff
@@ -357,7 +357,8 @@ def get_user_albums(self, user_id: int | None = None) -> PaginatedList[Album]:
         Get the favourites albums for the given user_id if provided or current user if not.
 
         :param user_id: the user ID to get favourites albums.
-        :return: a list of :class:`~deezer.Album` instances.
+        :returns: a :class:`~deezer.pagination.PaginatedList`
+                  of :class:`~deezer.Album` objects.
```
[the remark this answers is not in the document]
Comment (prndrbr) [reply src/deezer/client.py:361]: Thanks!

**Comment by prndrbr** [comment]
Applied suggestions and squashed

**Review by browniebroke — APPROVED** [review]

**Comment by browniebroke** [comment]
Thanks! 🎉🎉🎉

**Pull request merged**